docs/pages/guides/using-firebase.mdx

```diff
@@ -15,13 +18,14 @@ There are two different ways you can use Firebase in your projects:
 
 React Native supports both the native SDK and the JS SDK. The following sections will guide you through when to use which SDK and all the configuration steps required to use Firebase in your Expo projects.
 
-## Prerequisites
+### Prerequisites
```

Comment:
Not sure I understand the requirement to change the heading level here. I think all sub-sections headers should start in order.

Reply:
I just want to make it "a part" of the intro paragraph (which start with page title - `h1`), but have no strong opinion about this change, can revert that.

Reply:
In that case, you can remove the heading, "Prerequisites" altogether. I thought it'll be good to mention the obvious but I see your point as well 👍